src/SourceTerm/PointSource.cpp

```diff
@@ -98,8 +98,10 @@ void seissol::sourceterm::transformMomentTensor(
     }
   }
 
+#ifndef USE_ACOUSTIC
```

Comment:
Ok, may stay like that. I guess adjusting the point sources properly by material is something for the near future.